css/styles.css

```
@@ -0,0 +1,231 @@
+/* Main text color rgba(117, 117, 117, 1) */
+/* Headline text color rgba(33, 33, 33, 1) */
+/* Accent text color rgba(33, 150, 243, 1) */
+/* Logo text color rgba(0, 0, 0, 1)*/
+/* Footer text color rgba(255, 255, 255, 0.6)*/
+
+/* Main background color #F5F5F5 */
+/* Header background color rgba(255, 255, 255, 1) */
+/* Subsidiary background color rgba(245, 244, 250, 1) */
+/* Secondary background color rgba(47, 48, 58, 1) */
+
+
+/* Variables */
+:root {
+    --main-text-color: rgba(117, 117, 117, 1);
+    --headline-text-color: rgba(33, 33, 33, 1); 
+    --accent-text-color: rgba(33, 150, 243, 1); 
+    --logo-text-color: rgba(0, 0, 0, 1);
+    --footer-text-color: rgba(255, 255, 255, 0.6);
+    
+    --main-background-color: #F5F5F5; 
+    --header-background-color: rgba(255, 255, 255, 1); 
+    --subsidiary-background-color: rgba(245, 244, 250, 1); 
+    --secondary-background-color: rgba(47, 48, 58, 1); 
+
+}
+
+/* Common styles */
+body {
+    font-family: 'Roboto', sans-serif;
+
+    color: var(--main-text-color);
+    background-color: var(--main-background-color);
+}
+
+.logo,
+.active {
+    text-decoration: none;
+}
+
+.active {
+    cursor: pointer;
+}
+
+[class*="list"] {
+    list-style: none;
+}
```

WTF is this? Following is enough
```suggestion
ul {
    list-style: none;
}
```